src/routes_report.py

```diff
@@ -0,0 +1,28 @@
+import csv
+
+from src.report import Report
+import src.utils as utils
+
+class RoutesReport(Report):
+
+  def generate(self):
+    # Write fields to routes_report.csv 
+    self.writer.writerow(['timestamp', 'origin', 'destination'])
+
+    # Read routes.csv 
+    for index, row in enumerate(self.reader):
+      if index == 0:
+        continue
+      msg_json = utils.parse_msg_json(row[1])
+      origin, dest = self.get_origin_dest(msg_json)
```

Comment:
shouldn't this also be in `src.utils` since we might need to use `get_origin_dest` in other types of reports?

Reply:
will move it once that comes up